nncf/torch/quantization/layers.py

```diff
@@ -955,6 +958,9 @@ def set_levels(self):
         self.level_low, self.level_high = calculate_asymmetric_level_ranges(self.num_bits - scaled_num_bits)
 
     def quantize(self, x, execute_traced_op_as_identity: bool = False):
+        with DisableTorchFunction():
+            # in multi-device case after loading nncf checkpoint, quantizers have a different device.
+            self.to(x.device)
```

Comment:
Only as WA, need to be reworked before release, 
please create ticket and add TODO comment

Reply:
Added TODO